frontend/src/app/modules/header/header-item/header-item.component.ts

```diff
@@ -1,8 +1,20 @@
 import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { AuthService } from '@core/services/auth.service';
 
 @Component({
     selector: 'app-header-item',
     templateUrl: './header-item.component.html',
     styleUrls: ['./header-item.component.sass'],
 })
-export class HeaderItemComponent {}
+export class HeaderItemComponent {
+    // eslint-disable-next-line no-empty-function
+    constructor(private authService: AuthService, private router: Router) {}
+
+    public logout() {
+        return this.authService.signOut().then(() => {
+            localStorage.removeItem('user');
```

Comment:
Why we need duplication of localStorage.removeItem('user') there if signOut() already calls it?

Reply:
added accidentally. Already removed